cmd/limactl/main.go

```diff
@@ -14,19 +14,36 @@ import (
 	"github.com/mattn/go-isatty"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
 )
 
 const (
 	DefaultInstanceName = "default"
 )
 
 func main() {
+	if dir := os.Getenv("LIMA_GENMAN"); dir != "" {
```

Comment:
It was not supposed to be user-accessible, so didn't think it was necessary with a cmd

It would be possible to write the lima.1 from embedded markdown (with go-md2man)

Reply:
> It was not supposed to be user-accessible

You can just hide the command (see hostagent)